src/System.Windows.Forms/src/System/Windows/Forms/TreeNode.cs

```diff
@@ -226,6 +226,12 @@ public Color BackColor
             }
         }
 
+        /// <summary>
+        ///  This flag is necessary to avoid announcing the node status message
+        ///  during a bulk action (e.g. <see cref="TreeNode.ExpandAll"/> method)
+        /// </summary>
+        internal bool AnnouncementStopped { get; set; }
```

Comment:
```suggestion
        internal bool AnnouncementStopped { get; private set; }
```

I'd call it `IsAnnouncingNodeStatus`